plugins/modules/synchronize.py

```diff
@@ -636,15 +633,20 @@ def _write_password_to_pipe(proc):
     out_lines = out_clean.split('\n')
     while '' in out_lines:
         out_lines.remove('')
-    if module._diff:
-        diff = {'prepared': out_clean}
-        return module.exit_json(changed=changed, msg=out_clean,
-                                rc=rc, cmd=cmdstr, stdout_lines=out_lines,
-                                diff=diff)
 
-    return module.exit_json(changed=changed, msg=out_clean,
-                            rc=rc, cmd=cmdstr, stdout_lines=out_lines)
+    result = dict(changed=changed, rc=rc, cmd=cmdstr)
+
+    if quiet:
+        result['msg'] = "OUTPUT IS HIDDEN DUE TO 'quiet=true'"
+        result['stdout_lines'] = []
```

Comment:
I don't think this should be suppressed. `stdout_lines` doesn't get shown in the output, and having the data could be useful if someone wants to `register` the data into a variable and work with it. The problem described in #171 is just the message flooding the output.

Reply:
Would something like this work?

```
localhost | CHANGED => { 
    "changed": true,
    "cmd": "/usr/bin/rsync --delay-updates -F --compress --archive --out-format=<<CHANGED>>%i %n%L /home/tmp/src/path/ /home/tmp/dest/path/",
    "msg": "4 files/directories have been synchronized",
    "rc": 0,
    "stdout_lines": [
        ".d..t...... ./",
        ">f+++++++++ 1.txt",
        ">f+++++++++ 2.txt",
        ">f+++++++++ 3.txt",
        ">f+++++++++ 4.txt"
    ]
}
```